src/main/java/internet/market/dao/jdbc/ProductDaoJdbcImpl.java

```diff
@@ -0,0 +1,105 @@
+package internet.market.dao.jdbc;
+
+import internet.market.dao.ProductDao;
+import internet.market.exceptions.DataProcessingException;
+import internet.market.model.Product;
+import internet.market.util.ConnectionUtil;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+public class ProductDaoJdbcImpl implements ProductDao {
+
+    @Override
+    public Product create(Product item) {
+        String query = "INSERT INTO products (name, price) VALUES (?, ?) ";
+        try (Connection connection = ConnectionUtil.getConnection()) {
+            PreparedStatement statement = connection
+                    .prepareStatement(query, Statement.RETURN_GENERATED_KEYS);
+            statement.setString(1, item.getName());
+            statement.setDouble(2, item.getPrice());
+            statement.executeUpdate();
+            ResultSet resultSet = statement.getGeneratedKeys();
+            if (resultSet.next()) {
+                item.setId(resultSet.getLong(1));
+            }
+            return item;
+        } catch (SQLException e) {
+            throw new DataProcessingException("We can`t add product with id = " + item.getId(), e);
```

Comment:
fix this message according to comments from Bohdan

Reply:
done